### PR TITLE
task#6

### PR DESCRIPTION
**Pull request by OleksandrZheleznyi**

### CODE REVIEW

File: Methods2/Program.cs [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    {
+        static void Main(string[] args)
+        {
+            //2
+            int debt = 700;
+            Console.WriteLine("Your current debt is {0:#}", debt);
+            Console.WriteLine("Enter sum to pay off debt: ");
+            int money = Int32.Parse(Console.ReadLine());
+
+            Amortization(money, ref debt);
```
Comment (OleksandrIvanovTheBest) [inline Methods2/Program.cs]: GetAmortization

File: Methods2/Program.cs [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+                case "r":
+                    result = RemainderOfDivision(operand1, operand2);
+                    Console.WriteLine("{0} % {1} = {2:#}", operand1, operand2, result);
+                    break;
+                case "p":
+                    result = Pow(operand1, operand2);
+                    Console.WriteLine("{0} ^ {1} = {2:G}", operand1, operand2, result);
+                    break;
+                case "c":
+                    result = Concatenation(operand1, operand2);
```
Comment (OleksandrIvanovTheBest) [inline Methods2/Program.cs]: Concatenate

File: Methods2/Program.cs [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+                    Console.WriteLine("{0} + {1} = {2:#}", operand1, operand2, result);
+                    break;
+                case "d":
+                    result = Division(operand1, operand2);
```
Comment (OleksandrIvanovTheBest) [inline Methods2/Program.cs]: Divide

File: Methods2/Program.cs [hunk 4]
```diff
@@ -0,0 +1,130 @@
+﻿using System;
+
+namespace Methods2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //2
+            int debt = 700;
```
Comment (OleksandrIvanovTheBest) [inline Methods2/Program.cs]: use `decimal` for money values

File: Methods2/Program.cs [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+                default:
+                    Console.WriteLine("You entered another word");
+                    break;
+            }
+
+            //5(additional task)
+            double a = 5;
+            double b = 8;
+            double c = 34;
+            Calculate(ref a, ref b, ref c);
```
Comment (OleksandrIvanovTheBest) [inline Methods2/Program.cs:64]: `ref` is not suitable param type here
Think and write why?

**Review by OleksandrIvanovTheBest — CHANGES_REQUESTED** [review]

File: Methods2/Program.cs (hunk 5 above)
Comment (OleksandrZheleznyi) [reply Methods2/Program.cs:64]: I think param ref no need here.
Need to add to the function Console.WriteLine(a, b, c);

Comment (OleksandrIvanovTheBest) [reply Methods2/Program.cs:64]: I mean you need use `out` here. But why ?

Comment (OleksandrZheleznyi) [reply Methods2/Program.cs:64]: I try use out, but there was a mistake "use of unassigned out parameter"

Comment (OleksandrIvanovTheBest) [reply Methods2/Program.cs:64]: Disregard, leave `ref`

**Review by OleksandrIvanovTheBest — APPROVED** [review]

**Pull request merged**